app/Http/Controllers/CommentsController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Requests\CommentRequest; 
+use App\Models\Comment; 
+use Illuminate\Support\Facades\Auth;
+
+class CommentsController extends Controller
+{
+    public function store(CommentRequest $request,$id){
```

Comment:
こちらも `,` の後には空白を入れるようにしましょう　
```suggestion
    public function store(CommentRequest $request, $id){
```